src/lib/MVC/Symfony/Templating/RenderContentStrategy.php

```diff
@@ -45,7 +45,7 @@ public function render(ValueObject $valueObject, RenderOptions $options): string
 
         $renderer = $this->getFragmentRenderer($options->get('method', $this->defaultRenderer));
 
-        return $renderer->render($controllerReference, $currentRequest)->getContent();
+        return $renderer->render($controllerReference, $currentRequest, $options->has('params') ? ['params' => $options->get('params')] : [])->getContent();
```

Comment:
Readability: please split this into multiple lines.

Personally I'd just go with:
```suggestion
        return $renderer->render($controllerReference, $currentRequest, ['params' => $options->get('params', [])])->getContent();
```
and would've taken this into the account when processing further, effectively making params always present, reducing  cognitive complexity.
But no strong feelings here.